source/tutorials/metadata.rst

```diff
@@ -0,0 +1,141 @@
+
+Metadata merging is supported anywhere that metadata is accepted in QIIME 2. For example, it might be interesting to color an Emperor plot based on the study metadata, or the sample alpha diversities. This can be accomplished by providing both the sample metadata mapping file *and* the ``SampleData[AlphaDiversity]`` artifact:
+
+.. download::
+   :url: https://docs.qiime2.org/2017.6/data/tutorials/moving-pictures/core-metrics-results/unweighted_unifrac_pcoa_results.qza
+   :saveas: unweighted_unifrac_pcoa_results.qza
+
+.. command-block::
+   qiime emperor plot \
+     --i-pcoa unweighted_unifrac_pcoa_results.qza \
+     --m-metadata-file sample-metadata.tsv \
+     --m-metadata-file faith_pd_vector.qza \
+     --o-visualization unweighted-unifrac-emperor-with-alpha.qzv
+
+.. question::
+   What happens to the visualization if the order of the metadata files is reversed in the above ``emperor plot``? What about in the ``metadata tabulate`` example above?
+
+Exploring feature metadata
+--------------------------
+
+As noted above, metadata can exist for either axis of a ``FeatureTable``: the sample axis or the feature axis. The previous examples in this tutorial were focused on sample-oriented metadata.
+
+To get started with feature metadata, first download the example files:
+
+.. download::
+   :url: https://docs.qiime2.org/2017.6/data/tutorials/moving-pictures/rep-seqs.qza
+   :saveas: rep-seqs.qza
+
+.. download::
+   :url: https://docs.qiime2.org/2017.6/data/tutorials/moving-pictures/taxonomy.qza
+   :saveas: taxonomy.qza
+
+Next, merge the two metadata files, and ``tabulate`` them:
+
+.. command-block::
+   qiime metadata tabulate \
+     --m-input-file rep-seqs.qza \
+     --m-input-file taxonomy.qza \
+     --o-visualization tabulated-feature-metadata.qzv
+
+The resulting table shows the joined metadata files with a column of the the feature IDs, a column of the representative sequences, a column of the taxonomic assignments, and lastly, a column of the assignment confidence.
+
+.. question::
+   Are all QZA files metadata? Are all metadata files QZAs?
+
+Finally, there are export options available in the visualizations produced from ``metadata tabulate``. Using the results from ``tabulated-feature-metadata.qzv``, export the data as a new TSV. Open that file in a TSV viewer or text editor and note that the contents are the same as the interactive metadata table in the visualization.
+
+.. question::
+   Can the exported TSV from the above step be used as metadata? What are some benefits of being able to export metadata (hint: see the discussion above about metadata merging)? What about some potential drawbacks (hint: what happens to data provenance_ when data is exported from QIIME 2)?
```

Comment:
Related to this provenance question, can you briefly describe that artifact provenance is retained when viewing an artifact as metadata? That description would make sense in the section introducing "metadata-viewable artifacts". It'd be cool to suggest loading a viz in q2view to see the "metadata artifact's" provenance.